google_groups/groups/ci-team.yaml

```diff
@@ -8,6 +8,8 @@ spec:
   members:
   - email: andrey.velichkevich@gmail.com
     role: MEMBER
+  - email: alexey.volkov@ark-kun.com
```

Comment:
I'm approving first, if you find it inconvenient you can always update to your corporate email address too.